shimmingtoolbox/cli/b0map.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+
+import logging
+
+import click
+
+from .. import b0map
+
+@click.command()
+@click.option("--verbose", is_flag=True, help="Be more verbose.")
+@click.option("--coilshims", help="Output hardware-shim calibration.")
+@click.option("--pulseseq", help="Output pulse-sequence calibration.")
+@click.argument("input", type=click.Path(exists=True, file_okay=False))
+@click.argument("output", type=click.Path(exists=False))
+def main(verbose, coilshims, pulseseq, input, output):
```

Comment:
Shouldn't it be called another name since main is the normally called automatically. Shouldn't it be a specific function name and then called in the if __name == '__main__'?

Reply:
@gab-berest main() is never called automatically. It's only called automatically if you write:

```
if __name__ == '__main__':
    main()
```

Aside, that unfortunately suffers under python3's [import rule](https://www.python.org/dev/peps/pep-0328/#relative-imports-and-name), so it's sort of fallen out of favour, being replaced by setup.py's `console_scripts`:

```
entry_points={
  'console_scripts':
    {'b0map': 'shimmingtoolbox.cli.b0map:main'}
}
```

but again that's not automatic, that's explicitly telling python to call it.